src/better_lane_follow/include/LineDetect.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Created by: Raad Khan
+ * Created On: April 23, 2017
+ * Description: Takes in an image feed and generates lane lines.
+ * Usage:
+ * Subscribes to:
+ * Publishes to:
```

Comment:
I'm OK if we remove these two lines. It's likely that subscription and publish topics will change, and these will not, just causing confusion down the line.